### PR TITLE
chore: Localization of Superset pt. 1

### PR DESCRIPTION
**Pull request by artemonsh**
### SUMMARY
The localization of some components and especially charts is not complete yet. This PR is the effort to achieve the full localization of Superset. Additionally, it is a real pain to upgrade Superset as one should solve tons of merge conflicts, if they wrapped everything in `t()` and `__()` manually, while it is not reflected in the official repo :/

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by artemonsh** [comment]
@michael-s-molina could you take a look please?

File: superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx [hunk 1]
```diff
@@ -259,9 +259,11 @@ function AnnotationList({
             <span>{t('Annotation Layer %s', annotationLayerName)}</span>
             <span>
               {hasHistory ? (
-                <Link to="/annotationlayer/list/">Back to all</Link>
+                <Link to="/annotationlayermodelview/list/">
+                  {t('Back to all')}
+                </Link>
               ) : (
-                <a href="/annotationlayer/list/">Back to all</a>
+                <a href="/annotationlayermodelview/list/">{t('Back to all')}</a>
```
Comment (rusackas) [inline superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx:266]: Can you explain the URI changes here? Not sure if this is bycatch (explanation would be helpful), or somehow related to the translation effort.

File: superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx [hunk 2]
```diff
@@ -224,9 +224,12 @@ export class Tabs extends React.PureComponent {
       title: t('Delete dashboard tab?'),
       content: (
         <span>
-          Deleting a tab will remove all content within it. You may still
-          reverse this action with the <b>undo</b> button (cmd + z) until you
-          save your changes.
+          {t(
```
Comment (rusackas) [inline superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx:227]: I wonder if template literals would make this block a little more legible/parsable. Not a dealbreaker.

File: superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Scatter/Scatter.jsx [hunk 3]
```diff
@@ -36,14 +36,11 @@ function setTooltipContent(formData, verboseMap) {
     return (
       <div className="deckgl-tooltip">
         <TooltipRow
-          label={`${t('Longitude and Latitude')}: `}
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Scatter/Scatter.jsx:39]: I could see endless debate about translating colons and spaces... I'm not sure if I have strong feelings, but I can't help but wonder if strings like "Longitude and Latitude" might be recycled elsewhere, whereas they're not as reusable with the `: ` appended.

File: superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.jsx [hunk 4]
```diff
@@ -57,7 +58,7 @@ function setTooltipContent(formData) {
     return (
       <div className="deckgl-tooltip">
         {o.object.name && (
-          <TooltipRow label="name: " value={`${o.object.name}`} />
+          <TooltipRow label={t('name: ')} value={`${o.object.name}`} />
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.jsx:61]: Another example of what I commented on elsewhere. I would think `name` is already translated/used in numerous places, so translating things like this with punctuation/spacing kind of adds some overhead.

**Comment by rusackas** [comment]
This generally looks good to me! I added a couple clarifying questions. In general, there seems like only one real pint of contention, which is that I'm not sure we want to include the colon-space combo in the translated strings, as the word(s) without the punctuation may provide utility in other settings. For example, `name: ` is translated, but we might just want to leverage `name` which is _probably_ already translated. I think this applies to several instances. Curious if others with experience in translations (e.g. @geido @ktmud @etr2460) might feel differently.

**Comment by michael-s-molina** [comment]
@rusackas According to this [article](https://ux.shopify.com/how-to-concatenate-localized-strings-mindfully-bcc7b6d4961a) it seems sometimes the punctuation could also be translated. Check the **Punctuation** section.

**Review by rusackas — APPROVED** [review]

File: superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.jsx (hunk 4 above)
Comment (artemonsh) [reply superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.jsx]: Totally agree

File: superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx (hunk 1 above)
Comment (artemonsh) [reply superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx]: I don't know how it happened :( Changed back.

**Review by rusackas — APPROVED** [review]
Approving updates... trying to get CI to pass.

File: superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Arc/Arc.jsx [hunk 5]
```diff
@@ -37,11 +37,13 @@ function setTooltipContent(formData) {
   return o => (
     <div className="deckgl-tooltip">
       <TooltipRow
-        label={t('Start (Longitude, Latitude): ')}
+        // eslint-disable-next-line prefer-template
+        label={t('Start (Longitude, Latitude)' + ': ')}
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Arc/Arc.jsx]: I guess this _could_ be a template string, but I don't particularly mind this.

**Comment by rusackas** [comment]
Approved CI to run. Fingers crossed! 

Latest changes seem fine too. 

I'll circle back to merge this if CI passes. If it doesn't, nag me here or on Slack, and we'll sort it out!

**Pull request merged**